### PR TITLE
Loading the database within cantherm

### PR DESCRIPTION
**Pull request by mjohnson541**
This adds an input file block within cantherm that loads the RMG database.

### CODE REVIEW

File: rmgpy/cantherm/input.py [hunk 1]
```diff
@@ -75,7 +79,72 @@
...
+    if isinstance(transportLibraries, str): transportLibraries = [transportLibraries]
+    if isinstance(reactionLibraries, str): reactionLibraries = [reactionLibraries]
+    if isinstance(frequenciesLibraries, str): frequenciesLibraries = [frequenciesLibraries]
+    databaseDirectory = settings['database.directory']
+    thermoLibraries = thermoLibraries or []
+    transportLibraries = transportLibraries
+    # Modify reactionLibraries if the user didn't specify tuple input
+    if reactionLibraries:
+        index = 0
+        while index < len(reactionLibraries):
```
Comment (alongd) [inline rmgpy/cantherm/input.py]: Why not `for index in xrange(len(reactionLibraries))`?
(I understand that's the input in `rmgpy/input.py`)

File: rmgpy/cantherm/input.py [hunk 2]
```diff
@@ -75,7 +79,72 @@
...
+            kineticsFamilies = kineticsFamilies,
+            kineticsDepositories = kineticsDepositories,
+            depository = False, # Don't bother loading the depository information, as we don't use it
+        )
+    
+    for family in database.kinetics.families.values(): #load training
+        family.addKineticsRulesFromTrainingSet(thermoDatabase=database.thermo)
 
+    for family in database.kinetics.families.values():
+        family.fillKineticsRulesByAveragingUp(verbose=True)
```
Comment (alongd) [inline rmgpy/cantherm/input.py]: Any reason not to combine this with the above loop?
Also, perhaps consider setting `verbose=False`?

**Comment by alongd** [comment]
@mjohnson541, should we also load all transport libraries (and groups) by default as part of this PR?

**Comment by mjohnson541** [comment]
contained in #1393

**Pull request closed without merging**